crates/turborepo-repository/src/change_mapper/package.rs

```diff
@@ -115,7 +115,7 @@ impl PackageChangeMapper for GlobalDepsPackageChangeMapper<'_> {
         // workspace package dependencies
         if matches!(
             path.as_str(),
-            "package.json" | "pnpm-lock.yaml" | "yarn.lock"
+            "package-lock.json" | "pnpm-lock.yaml" | "yarn.lock"
```

Comment:
Not necessary for this PR, but we should switch this to `PackageManager::supported_managers().iter().any(|pm| pm.lockfile_name() == path.as_str())` to avoid mistyping/missing lockfiles

Reply:
great: moved to https://github.com/vercel/turborepo/pull/9957/files and removed from this PR in eab7f354235027c6959ea6f729db67cc64964a02